onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -696,9 +699,10 @@ public float getZoomLevelAsFloat() {
     }
 
     @Override
-    public void setRouteOverlay(int lineOverlayColor, ObaShape[] shapes) {
+    public void setRouteOverlay(int lineOverlayColor, ObaShape[] shapes, boolean clear) {
         if (mMap != null) {
-            mLineOverlay.clear();
+            if (clear)
```

Comment:
Please always use brackets `{` for IF statements, even if it's just a one-liner.